ingest/ingest_files.py

```diff
@@ -10,32 +10,39 @@
 import re
 from itertools import islice
 
+import pandas as pd
+
 
 class IngestFiles:
-    def __init__(self, file_path, allowed_file_types, is_MTX=False):
+    def __init__(self, file_path, allowed_file_types, *, is_MTX=False, open_as=None):
         if not os.path.exists(file_path):
             raise IOError(f"File '{file_path}' not found")
         self.allowed_file_types = allowed_file_types
-        self.file_type, self.file = self.open_file(file_path)
+        self.file_type, self.file = self.open_file(file_path, open_as)
+        print(self.file_type)
```

Comment:
Again, is there a reason for this print statement?

Reply:
Nope. Addressed in 6f04851